accelerator/mqtt_main.c

```diff
@@ -0,0 +1,43 @@
+#include "config.h"
+#include "errors.h"
+#include "mqtt_utils/client_common.h"
+#include "mqtt_utils/duplex_callback.h"
+#include "mqtt_utils/duplex_utils.h"
```

Comment:
I dislike the file name `accelerator/mqtt_main.c`, which is NOT one of the functional accelerators.